framework/math/SpatialDiscretization/CellMappings/cell_mapping_base.cc

```diff
@@ -6,12 +6,13 @@
 
 #include "mesh/MeshContinuum/chi_meshcontinuum.h"
 
-chi_math::CellMapping::
-  CellMapping(const chi_mesh::MeshContinuum& in_grid,
-              const chi_mesh::Cell& in_cell,
-              size_t in_num_nodes,
-              std::vector<std::vector<int>> in_face_node_mappings,
-              const VandAFunction& volume_area_function) : ref_grid_(in_grid),
+chi_math::CellMapping::CellMapping(
+  const chi_mesh::MeshContinuum& in_grid,
+  const chi_mesh::Cell& in_cell,
+  size_t in_num_nodes,
+  std::vector<std::vector<int>> in_face_node_mappings,
```

Comment:
We should move away from these `in_` arguments as we encounter them. These are not necessary now that members use a trailing underscore.

Reply:
Looks like I accidentally hit the reformat short-cut on this file. I might have added something then removed it.

I agree with your comment, and take responsibility for it, however, this can send this PR down a rabbit hole and changed into a "fix the in_ business". I would love help in committing a single PR with just formatting changes.